picard/script.py

```diff
@@ -988,7 +1076,8 @@ def func_get_multi(parser, multi, item_index, separator=MULTI_VALUED_JOINER):
         return ''
 
 
-def func_foreach_multi(parser,
+@script_function(eval_args=False)
+def func_foreachmulti(parser,
```

Comment:
I think `foreach`, `while` and `map` are fine. `foreachmulti` looks strange and the `multi` is not really needed here.

Reply:
Well, since those only apply to "multi", I just kept the style of `inmulti` and `lenmulti`.

Reply:
Thanks for the change, I think that reads better.

IMHO those names are also awkward :D But were needed for distinguishing from existing `$in` and `$len`. But maybe we should have used actually different names, e.g. `contains` and `count`. Not sure about the first one, but `count` sounds actually much better to me then `lenmulti`. Maybe we should consider to add an alias for this (and deprecate `$lenmulti`), what do you think?

Reply:
Yup, we could create aliases and deprecate, but imho we'll have to keep `lenmulti` and `inmulti` for a long time... not sure it is worth the hassle.